thunder/torch/__init__.py

```diff
@@ -4070,7 +4070,13 @@ def sigmoid(a: TensorLike, /) -> TensorLike:
 
 # CompositeImplicitAutograd - don't register decomp
 @torchsymbol(torch.softmax, torch.nn.functional.softmax, is_method=True)
-def softmax(a: TensorLike, /, dim: int, *, dtype: None | dtypeLike = None) -> TensorLike:
+def _softmax(
```

Comment:
With apologies, @crcrpar, I think my suggestion was slightly wrong. I think the `torchsymbol` decorator now has to set the `id` explicitly, otherwise the name `_softmax` is used

Reply:
I think the registration is a little more complicated. There are actually places we use `fn.__name__` directly in:

https://github.com/Lightning-AI/lightning-thunder/blob/cfc9ea747aac5e7e64279b7a5d07cf543aa1ab1a/thunder/torch/__init__.py#L73

I think we need to add an override for the `fn.__name__` lookup so we see "softmax" instead of "_softmax"